code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -390,6 +390,8 @@
 									span_notice("You cut \the [src] apart with \the [W]."))
 				deconstruct(TRUE)
 				return
+		if (user.combat_mode)
+			return FALSE
```

Comment:
this should happen after the welding check for consistency with other weldable stuff.

Reply:
it does, the weldercheck below only happens if the locker is closed. the if statement above this is the welder check for being open